src/main/automation/ClickService.ts

```
@@ -0,0 +1,186 @@
+import { Debugger } from 'electron';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)